guides/v2.2/javascript-dev-guide/widgets/widget_dropdown.md

```diff
@@ -0,0 +1,202 @@
+    @_dropdown-list-border: @dropdown-list__border,
+    @_dropdown-list-pointer: @dropdown-list__pointer,
+    @_dropdown-list-pointer-border: @dropdown-list-pointer__border,
+    @_dropdown-list-pointer-position: @dropdown-list-pointer__position,
+    @_dropdown-list-pointer-position-top: @dropdown-list-pointer__position-top,
+    @_dropdown-list-pointer-position-left-right: @dropdown-list-pointer__position-left-right,
+    @_dropdown-list-item-border: @dropdown-list-item__border,
+    @_dropdown-list-item-padding: @dropdown-list-item__padding,
+    @_dropdown-list-item-margin: @dropdown-list-item__margin,
+    @_dropdown-list-item-hover: @dropdown-list-item__hover,
+    @_dropdown-list-shadow: @dropdown-list__shadow,
+    @_dropdown-list-z-index: @dropdown-list__z-index,
+
+    @_dropdown-toggle-icon-content: @dropdown-toggle-icon__content,
+    @_dropdown-toggle-active-icon-content: @dropdown-toggle-icon__active__content,
+
+    @_icon-font: @dropdown-toggle-icon__font,
+    @_icon-font-size: @dropdown-toggle-icon__font-size,
+    @_icon-font-line-height: @dropdown-toggle-icon__font-line-height,
+    @_icon-font-color: @dropdown-toggle-icon__font-color,
+    @_icon-font-color-hover: @dropdown-toggle-icon__font-color-hover,
+    @_icon-font-color-active: @dropdown-toggle-icon__font-color-active,
+    @_icon-font-margin: @dropdown-toggle-icon__font-margin,
+    @_icon-font-position: @dropdown-toggle-icon__position,
+    @_icon-font-vertical-align: @dropdown-toggle-icon__font-vertical-align,
+    @_icon-font-text-hide: @dropdown-toggle-icon__text-hide
+) {
+    // ... 
+}
+```
+
+[LESS mixin `.lib-dropdown()`]: {{ site.mage2bloburl }}/{{ page.guide_version }}/lib/web/css/source/lib/_dropdowns.less
+
+## Initialize the dropdown widget {#dropdown_init}
+
+The loader widget is initialized as described in [JavaScript initialization].
+
+### Initialize with `data-mage-init` attribute
+
+```html
+<div class="magento__dropdown-widget">
+    <span class="action toggle"
+          data-toggle="dropdown"
+          aria-haspopup="true"
+          data-mage-init='{"dropdown":{}}'>
+        <span>Dropdown Opener Button</span>
+     </span>
+     <ul class="dropdown-options" data-target="dropdown" />
+</div>
+```
+
+### Initialize in `.js` file with options
+
+```js
+$('.magento__dropdown-widget').dropdown();
+```
+
+## Options {#dropdown_options}
+
+-  [parent](#d_parent)
+-  [autoclose](#d_autoclose)
+-  [btnArrow](#d_btnArrow)
+-  [menu](#d_menu)
+-  [activeClass](#d_activeClass)
+
+Description of each option as follows below location.
+
+### `parent`   {#d_parent}
+
+The parent of element on which the widget was initialized. If not specified,
+the widget finds it himself with jQuery method `.parent()`.
+
+**Type**:
+
+- jQuery object
+- HTML
+- String
+
+**Default value**: `null`
+
+
+### `autoclose`   {#d_autoclose}
+
+Whenever to close or not the dropdown menu when mouse is clicked outside of dropdown scope.
```

Comment:
Please, adjust this part. We don't want to abuse people who use a touchpad, do we? :)
```suggestion
Whenever to close or not the dropdown menu when the click is performed outside of dropdown scope.
```